docs/migrating-to-tool-kit.md

```diff
@@ -227,18 +229,39 @@ hook for a `test` target, for example,) but sometimes you'll want to leave some
 of the targets in at the beginning of the migration and gradually transition
 everything over to Tool Kit. Regardless, this step just provides some
 suggestions and the migration tool doesn't perform any actions so you'll need to
-delete the Makefile yourself if you don't think it's needed anymore. Remember
-that – assuming you didn't delete it in the earlier step – some of the `make`
+delete the Makefile yourself if you don't think it's needed anymore.
+
+Remember that – assuming you didn't delete it in the earlier step – some of the `make`
 commands come from `n-gage` itself and won't necessarily be in the Makefile,
 such as `make install`.
 
+You'll also want to ensure that:
+
+- You use the same node entry point of the app on `npm start`
+- You pull through any env set in the Makefile to an .rc file eg. for tests
```

Comment:
**suggestion**: I don't really follow what either of these tips entail, maybe you could include an example of what the change would look like for both?